.entity-processor-json.py

```diff
@@ -16,7 +16,7 @@ def codePointToString(codePoint):
   return string
 
 # this uses 658 MB
-document = xml.dom.minidom.parse('%s/unicode.xml' % os.environ['HTML_CACHE'])
+document = xml.dom.minidom.parse('%s/unicode.xml' % os.environ['HTML_TEMP'])
```

Comment:
Future cleanup? I think we should also rename these files to not have a dot at the start...